x-pack/legacy/plugins/triggers_actions_ui/np_ready/public/application/components/builtin_action_types/server_log.test.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
+import { TypeRegistry } from '../../type_registry';
+import { registerBuiltInActionTypes } from './index';
+import { ActionTypeModel, ActionConnector } from '../../../types';
+
+const ACTION_TYPE_ID = '.server-log';
+let actionTypeModel: ActionTypeModel;
+
+beforeAll(() => {
+  const actionTypeRegistry = new TypeRegistry<ActionTypeModel>();
+  registerBuiltInActionTypes({ actionTypeRegistry });
+  const getResult = actionTypeRegistry.get(ACTION_TYPE_ID);
+  if (getResult !== null) {
+    actionTypeModel = getResult;
+  }
+});
+
+describe('actionTypeRegistry.get() works', () => {
+  test('action type static data is as expected', () => {
+    expect(actionTypeModel.id).toEqual(ACTION_TYPE_ID);
+    expect(actionTypeModel.iconClass).toEqual('logsApp');
+  });
+});
+
+describe('server-log connector validation', () => {
+  test('connector validation succeeds when connector config is valid', () => {
+    const actionConnector = {
+      secrets: {},
+      id: 'test',
+      actionTypeId: '.server-log',
+      name: 'server-log',
+      config: {},
+    } as ActionConnector;
+
+    expect(actionTypeModel.validateConnector(actionConnector)).toEqual({
+      errors: {},
+    });
+  });
+});
+
+describe('action params validation', () => {
+  test('action params validation succeeds when action params is valid', () => {
+    const actionParams = {
+      message: 'test message',
+      level: 'trace',
+    };
+
+    expect(actionTypeModel.validateParams(actionParams)).toEqual({
+      errors: {},
+    });
+  });
```

Comment:
I didn't complete this part of validation actions (and test coverage), because it is a part of Create Alert form. But if you mentioned this - I will do